app/models/container_project.rb

```diff
@@ -10,22 +10,22 @@ class ContainerProject < ApplicationRecord
   include Purging
   belongs_to :ext_management_system, :class_name => "ManageIQ::Providers::ContainerManager", :foreign_key => "ems_id", :inverse_of => :container_projects
   has_many :container_groups, -> { active }, :inverse_of => :container_project
-  has_many :container_routes
-  has_many :container_replicators
-  has_many :container_services
+  has_many :container_routes, :dependent => :destroy
+  has_many :container_replicators, :dependent => :destroy
+  has_many :container_services, :dependent => :destroy
```

Comment:
See ☝️ registries: https://github.com/ManageIQ/manageiq/pull/23389/files#r2007811459

There are hundreds of these to hundreds projects.  Low churn or handled elsewhere?

Reply:
Same here